tests/Pyxsim/__init__.py

```diff
@@ -0,0 +1,134 @@
+
```

Comment:
Needs a copyright comment and a comment referencing the license.  See the Source check failures in the Jenkins run.

Reply:
added

Reply:
I will fix on #165 if that's acceptible

Reply:
actually these are already fixed..

Reply:
Cool